author.rb

```diff
@@ -0,0 +1,18 @@
+require 'securerandom'
+
```

Comment:
Great to add all Functionalities for the author classes but I would like to suggest adding an import for items class it will be helpful for testing time 😄